lib/network_utils.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Functional methods to operate on network
+# Maintainer: Rodion Iafarov <riafarov@suse.com>
+package network_utils;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use testapi;
+
+our @EXPORT = qw(setup_static_network recover_network can_upload_logs);
+
+=head2 setup_static_network
+Configure static IP on SUT with setting up default GW.
+Also doing test ping to 10.0.2.2 to check that network is alive
+=cut
+sub setup_static_network {
+    my ($ip, $gw) = @_;
```

Comment:
die when no ip specified or provide default value

Reply:
Fixed, thanks for the review!